### DISCUSSION TITLE
sonoff bridge RF

### DISCUSSION THREAD

**Discussion by sihamster** [indomus/forum#252] [Nuove idee, proposte, progetti e guide]
Ciao tutti,

in uno scenario di rilevazione acqua utilizzavo un sonoff bridge RF in pairing con diversi rilevatori di acqua RF  433 Mhz e un sonoff basic r1. Quest' ultimo ha smesso di funzionare.
Dovendo acquistare un nuovo attuatore in sostituzione del basic r1, volevo spostare il "progetto" sul protocollo zigbee.
Stavo attenzionando il nuovo sonoff basic r3 RF e il recente sonoff bridge zigbee, ma quest' ultimo non mi sembra implementi l' RF.
Avete dei suggerimenti per ricostruire lo scenario in ambito zigbee?

**Comment by Scialla** [comment]
Non mi risulta esistano bridge RF compatibili con zigbee.. (per ora), ma magari mi sbaglio

**Reply by sihamster** [reply]
Beh è una prima conferma.
Sai se sono commercializzati sensori di rilevamento acqua che comunicano con il recente [sonoff bridge zigbee](https://www.itead.cc/wireless/bluetooth-and-zigbe/sonoff-zbbridge.html)?
Ripeto, vorrei spostare la rilevazione su protocollo zigbee anche in previsione di aumento di sensori

**Reply by Scialla** [reply]
sensori per rilevare le perdite d'acqua? Se si ce ne sono diversi zigbee, così come per il fumo, la CO2.. 
Tieni presente che RF ha una portata di 100-150 mt, zigbee è molto ridotto in confronto e, il sonoff Bridge Zigbee, ha portata veramente infima, ci sono soluzioni molto più performanti, coordinator che funzionano molto meglio, basati sul chip 2538 o 2652 e amplificatori. 
Ti consiglio poi di appoggiarti a zigbee2mqtt piuttosto che ad altre soluzioni, è la più semplice e funzionale, oltre che la più supportata, mentre con il sonoffbridge dovresti usare zigbee2tasmota (potente ma complicato da usare) o ZHA (semplice ma supporta meno device rispetto a z2m)
Vero è che zigbee crea una rete mesh grazie a device alimentati a corrente che fanno da ripetitori (router) ma, se non arriva il segnale, devi calcolare di aggiungere device in mezzo per estendere ed arrivare dove ti serve.

**Reply by sihamster** [reply]
grazie delle indicazioni. Ho iniziato a vedere le potenzialità del 2538 con ampli. Hai dei rif.ti per il sensore antiallagamento?

**Reply by Scialla** [reply]
cerca su aliexpress sensore acqua aqara

**Reply by snakuzzo** [reply]
@sihamster ricorda che quello che leggi sulla portata è tutto TEORICO. Ci sono decine di variabili sulla copertura. Ti dico che per com'è strutturata casa mia (60mq scarsi) faccio fatica a coprire tutto se piazzo il coordinator ad un estremo ed alcuni end device all'altro estremo.
Questo può dipendere da interferenze sulla 2.4GHz (quindi tutti i vari AP Wifi in zona), pareti, etc.
Con CC2538+92 (che però è considerato legacy da TI e quindi ha firmware solo 3.0 massimo) e CC2562 (io uso questo attualmente e supporta i nuovi firmware 3.x) non dovresti avere particolari problemi...soprattutto se valuterai di renderlo indipendente dal tuo server su cui fai girare il tuo hub. A tal proposito ci sono diverse soluzioni
